runners/direct-java/src/test/java/org/apache/beam/runners/direct/DirectRunnerTest.java

```diff
@@ -109,10 +112,12 @@ public String apply(KV<String, Long> input) {
     result.awaitCompletion();
   }
 
+  private static AtomicInteger changed;
```

Comment:
use a local final variable in the method to prevent other tests from accessing it